### ISSUE TITLE
Dev is lazy

### ISSUE THREAD

**Issue by Courageousodin** [ActuallyOdin/OdinBot#6]
i gave him a list of things i wanted for the bot but its been like half a year and none of it is done. Recommend replacement dev. maybe tsarcasm.

**Comment by ShatteredPlayz** [comment]
I agree tsarcasm would be a good replacement, also this Joshie guy, who is really good at server hosting and managing.

**Comment by ShatteredPlayz** [comment]
rude :/
